TTBot/Models/ChampionshipAliasesModel.cs

```diff
@@ -0,0 +1,48 @@
+﻿using ServiceStack.Text;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace TTBot.Models
+{
+    public class ChampionshipAliasesModel
+    {
+        private static Dictionary<string, string[]> aliasesDictionary = new Dictionary<string, string[]>() {
+            { "ACC GT3 8pm", new string[]{ "ACC Pro", "Pro Am", "Pro/Am", "GT Pro", "Mon Pro", "Monday Pro" } },
```

Comment:
I'd like to move this config to the DB at some point if possible, so we can avoid code changes when theres new championships.

Reply:
As per other comment.